msteams-platform/bots/how-to/Teams conversational AI/conversation-ai-quick-start.md

```diff
@@ -20,12 +20,12 @@ To get started, ensure that you have the following tools:
 | Install | For using... |
 | --- | --- |
 | &nbsp; | &nbsp; |
-| [Visual Studio Code](https://code.visualstudio.com/download) or  [Visual Studio](https://visualstudio.microsoft.com/downloads/) | JavaScript, TypeScript, or CSharp build environments. Use the latest version. |
+| [Visual Studio Code](https://code.visualstudio.com/download) or  [Visual Studio](https://visualstudio.microsoft.com/downloads/) | JavaScript, TypeScript, Python, or CSharp build environments. Use the latest version. |
```

Comment:
For Python tab, can we also please add these prereqs:
- Python (>=3.8, <4.0)
- Poetry
- Python VSCode Extension

Descriptions and links found [here ](https://github.com/microsoft/teams-ai/blob/main/getting-started/QUICKSTART.md#python-quickstart)

Reply:
Updated.